pkg/ddevapp/settings_test.go

```diff
@@ -41,7 +42,7 @@ func TestWriteSettings(t *testing.T) {
 		"drupal6":   "sites/default/settings.ddev.php",
 		"drupal7":   "sites/default/settings.ddev.php",
 		"drupal8":   "sites/default/settings.ddev.php",
-		"wordpress": "wp-config.php",
+		"wordpress": "wp-config-ddev.php",
```

Comment:
Yeah, I think this is where things got backwards. If I'm reading right the settings path should be wp-config.php.

Reply:
The reasoning behind the original naming of "local" and such: "settings.local.php" is the traditional name of a site-specific add-on for a drupal settings file. So D8 settings.php has a commented out "load settings.local.php" stanza. When we switched to settings.ddev.php, the name "local" followed, which is now quite confusing because people are referring to ddev as "local" or DDEV-Local, but that use of "local" has nothing to do with what was going on here.